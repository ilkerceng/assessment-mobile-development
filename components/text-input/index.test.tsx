import React from "react";
import renderer from "react-test-renderer";
import TextInput from "./";

describe("<text-input />", () => {
  test("match snapshot", () => {
    const tree = renderer.create(<TextInput onChange={() => false} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
