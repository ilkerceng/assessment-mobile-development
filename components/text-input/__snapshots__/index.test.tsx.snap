// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<text-input /> match snapshot 1`] = `
<TextInput
  allowFontScaling={true}
  onChangeText={[Function]}
  rejectResponderTermination={true}
  style={
    Object {
      "borderColor": "gray",
      "borderWidth": 1,
      "height": 40,
    }
  }
  underlineColorAndroid="transparent"
  value=""
/>
`;
